src/runner.ts

```diff
@@ -12,43 +13,43 @@ const delay = delayInms => {
     }, delayInms);
   });
 };
-export const runnerFunction = async (options: {
-  list: string;
-  html: string;
-  text: string;
-  subject: string;
-  email: string;
-}) => {
+export const runnerFunction = async (options: { list: string; html: string; text: string; subject: string }) => {
   try {
     console.log(chalk.blue(chalk.bold('Info ')) + 'Reading HTML File...');
     let htmlBuffer = await fs.readFile(join(__dirname, '..', options.html));
     console.log(chalk.blue(chalk.bold('Info ')) + 'Reading Text File...');
     let textBuffer = await fs.readFile(join(__dirname, '..', options.text));
     console.log(chalk.blue(chalk.bold('Info ')) + 'Fetching Mailing List...');
-    let response = await DatabaseService.getMongoDatabase().collection('alexa-mail').findOne({ name: options.list });
+    let response = await DatabaseService.getMongoDatabase()
+      .collection(process.env.COLLECTION_NAME)
```

Comment:
any specific reason behind keeping the collection name in env?

Reply:
This will allow this repository, to not be restricted to one community and be more generalised so everyone can use it easily.